src/services/inlayHints.ts

```diff
@@ -736,6 +737,8 @@ export function provideInlayHints(context: InlayHintsContext): InlayHint[] {
                         visitDisplayPart(propertySignature.type);
                     }
                     break;
+                default:
+                    Debug.fail(`Unhandled node kind: ${node.kind}`);

```

Comment:
There's actually a helper for this one which will print better.
```suggestion
                    Debug.failBadSyntaxKind(node);
```